### PR TITLE
Update logging-control.md

### PR DESCRIPTION
**Pull request by kchristidis**
## Description

Update references to `core.yaml`.

(Changed the header levels, and removed some double spaces after periods while at it as well.)
## How Has This Been Tested?

No tests necessary.
## Checklist:

- [x] I have added a [Signed-off-by](https://github.com/hyperledger/fabric/blob/master/CONTRIBUTING.md#legal-stuff)
- [x] Either no new documentation is required by this change, OR I added new documentation
- [x] Either no new tests are required by this change, OR I added new tests
- [] I have run [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports), [go vet](https://golang.org/cmd/vet/), and [golint](https://github.com/golang/lint). I have cleaned up all valid errors and warnings in code I have added or modified. These tools may generate false positives. Don't be worried about ignoring some errors or warnings. The goal is clean, consistent, and readable code.

Signed-off-by: Kostas Christidis kchristidis@us.ibm.com

### CODE REVIEW

File: docs/dev-setup/logging-control.md [hunk 1]
```diff
@@ -1,78 +1,52 @@
...
-The logging level of the `peer` command can be controlled from the command
-line for each invocation using the `--logging-level` flag, for example
+The logging level of the `peer` command can be controlled from the command line for each invocation using the `--logging-level` flag, for example
 
     peer node start --logging-level=debug
 	
-The default logging level for each individual `peer` subcommand can also
-be set in the **openchain.yaml** file. For example the key `logging.peer` sets
-the default level for the `peer` subcommmand.
+The default logging level for each individual `peer` subcommand can also be set in the `core.yaml` file. For example the key `logging.peer` sets the default level for the `peer` subcommmand.
```
Comment (srderson) [inline docs/dev-setup/logging-control.md]: Maybe link to `core.yaml`?

Comment (kchristidis) [inline docs/dev-setup/logging-control.md]: I can certainly do that, but I'm all for consistency and I don't think we link to files in our documentation in general. Do you want this to be the default policy from now on? If so, LMK and I'll edit to add the link.

Comment (srderson) [inline docs/dev-setup/logging-control.md]: Some docs such as https://github.com/hyperledger/fabric/blob/master/docs/API/SandboxSetup.md link to files from the doc. I think it's useful and should be the default policy, but open to other opinions.

Comment (kchristidis) [inline docs/dev-setup/logging-control.md]: I had missed that. Will add the links now then.

Comment (kchristidis) [inline docs/dev-setup/logging-control.md]: @srderson: Done.

**Comment by srderson** [comment]
Thanks @kchristidis! LGTM

**Pull request merged**